CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Features
 
+
+- [#251](https://github.com/pybop-team/PyBOP/pull/251) - Increment PyBaMM > v25.3, remove redundant tests within integration tests, increment citation version, fix examples with incorrect model definitions.
```

Comment:
```suggestion
- [#251](https://github.com/pybop-team/PyBOP/pull/251) - Increment PyBaMM > v23.5, remove redundant tests within integration tests, increment citation version, fix examples with incorrect model definitions.
```

Reply:
Ha, nice catch! That would be an interesting bug for someone looking back in a few years 😄